src/tests/CustomerRouter.test.ts

```diff
@@ -0,0 +1,12 @@
+import { customContainer } from "../di"
+import { CustomerRouter } from "../presentation/routers/CustomerRouter"
+
+describe('CustomerRouter', () => {
+    it('Should return a router when setup is called', () => {
+        const costumerRouter = customContainer.resolve(CustomerRouter);
```

Comment:
O nome correto não seria customerRouter?
```suggestion
        const customerRouter = customContainer.resolve(CustomerRouter);
```